TwitterKit/TwitterKit.xcodeproj/project.pbxproj

```diff
@@ -764,6 +766,8 @@
 		22BA0E6E192560F400A9F03E /* TWTRPersistentStoreTest.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = TWTRPersistentStoreTest.m; sourceTree = "<group>"; };
 		2908077F1D4ABB9800CFBB6E /* TWTRTimelineViewControllerDelegateTests.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = TWTRTimelineViewControllerDelegateTests.m; sourceTree = "<group>"; };
 		2B4EB31824F9F499003D6CEE /* CoreServices.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = CoreServices.framework; path = System/Library/Frameworks/CoreServices.framework; sourceTree = SDKROOT; };
+		2B500A7D24FBF18500EEE584 /* WebKit.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = WebKit.framework; path = System/iOSSupport/System/Library/Frameworks/WebKit.framework; sourceTree = SDKROOT; };
+		2B500A8224FC20AE00EEE584 /* TWTRTestCase.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; name = TWTRTestCase.h; path = "../../../../GeneXus/GitHub/twitter-kit-ios/shared/TwitterTestFoundation/Base/TWTRTestCase.h"; sourceTree = "<group>"; };
```

Comment:
Is this path OK or it's specific to you machine?

Reply:
Fix in 5839f61